schemas/settings.json

```diff
@@ -66,9 +66,22 @@
             "minLength": 1,
             "default": "lib"
         },
+        "commands_dir": {
+            "title": "commands dir",
+            "description": "A directory to nested command files of the current script",
+            "oneOf": [
```

Comment:
I guess it's better to put schemas containing `"type": "null"` at the end, not to introduce inconsistency. But overall this change looks nice. ;)